schemas/test_cases/v0/experiment.yaml

```diff
@@ -32,9 +32,9 @@
       environment_variables: {}
       force_pull_image: false
       image:
-        cpu: determinedai/environments:py-3.8-pytorch-1.12-tf-2.8-cpu-6218891
-        cuda: determinedai/environments:cuda-11.3-pytorch-1.12-tf-2.8-gpu-6218891
-        rocm: determinedai/environments:rocm-5.0-pytorch-1.10-tf-2.7-rocm-6218891
+        cpu: determinedai/environments-dev:py-3.8-pytorch-1.12-tf-2.12-cpu-0.21.2
+        cuda: determinedai/environments-dev:cuda-11.3-pytorch-1.12-tf-2.12-gpu-0.21.2
+        rocm: determinedai/environments:rocm-5.0-pytorch-1.10-tf-2.7-rocm-9d07809
```

Comment:
question: Just for my own chance to learn, could you tell me what this change to `rocm` is for?

Reply:
Oops, that is a mistake! Dunno how that happened - overzealous pasting, I suppose.

Reply:
This change will be reverted before I merge, though, so it's not a worry.

Reply:
Leaving this open because there's something I think I don't understand about the release plan, but that may become obvious shortly.

Reply:
It became obvious. I hadn't understood that the PR was a WIP you were going to change before committing.